src/package.json

```diff
@@ -45,30 +47,45 @@
     "jsonwebtoken": "^8.5.1",
     "lodash.throttle": "^4.1.1",
     "node-fetch": "^2.6.0",
-    "nuxt": "^2.11.0",
+    "nuxt": "^2.15.7",
     "nuxt-svg-loader": "^1.2.0",
     "portal-vue": "^1.5.1",
+    "postcss": "^8.4.14",
     "resize-observer": "^1.0.0",
     "stripe": "^8.111.0",
     "swiper": "^4.5.0",
     "tiny-cookie": "^1.0.1",
     "url-parse": "^1.4.7",
+    "uuid": "^8.3.2",
     "v-tooltip": "^2.0.2",
+    "vue": "^2.6.8",
+    "vue-class-component": "^7.2.6",
     "vue-clipboard2": "^0.3.1",
     "vue-cookie": "^1.1.4",
-    "vue-gtag": "^1.10.0",
+    "vue-gtag": "^1.16.1",
     "vue-i18n": "^8.9.0",
     "vue-lazyload": "^1.2.6",
+    "vue-property-decorator": "^9.1.2",
+    "vue-server-renderer": "^2.6.14",
     "vue-vimeo-player": "0.0.9",
-    "vuex": "^3.1.2"
+    "vuex": "^3.1.2",
+    "vuex-class": "^0.3.2"
```

Comment:
`vue` and `vue-server-renderer` (and vue-template-compiler)should have exact same version
in fact I usually just don't declare them in package.json and let them obey nuxt defined versions